docs/partitioning.txt

```diff
@@ -22,17 +21,15 @@ Explicit graph partitioning can only enabled against storage backends that suppo
 
 There are two aspects to graph partitioning which can be individually controlled: edge cuts and vertex cuts.
 
-Edge Cut
-~~~~~~~~
+=== Edge Cut
```

Comment:
Let me clarify what I think needs to be done. I am not discouraging edge cuts, just informing the user that they make more sense when there is locality of purpose (storage and compute). I indicated below the place to insert the sentence.